jib-build-plan/src/test/java/com/google/cloud/tools/jib/api/buildplan/ContainerBuildPlanTest.java

```diff
@@ -121,10 +127,12 @@ private ContainerBuildPlan createSamplePlan() {
         FileEntriesLayer.builder()
             .addEntry(Paths.get("/src/file/foo"), AbsoluteUnixPath.get("/path/in/container"))
             .build();
+    Platform platform = new Platform("os1", "arch1");
     return ContainerBuildPlan.builder()
         .setBaseImage("base/image")
         .setArchitectureHint("arch")
         .setOsHint("os")
+        .setPlatforms(Arrays.asList(platform))
```

Comment:
nit: let's just inline this.
```suggestion
        .setPlatforms(Arrays.asList(new Platform("os1", "arch1")))
```